SPECS/net-snmp/net-snmp.spec

```diff
@@ -1,8 +1,8 @@
 %global __requires_exclude perl\\(.*\\)
 Summary:        Net-SNMP is a suite of applications used to implement SNMP v1, SNMP v2c and SNMP v3 using both IPv4 and IPv6.
 Name:           net-snmp
-Version:        5.9
-Release:        6%{?dist}
+Version:        5.9.1
+Release:        1%{?dist}
```

Comment:
When I tried building it locally, the build failed. 
Adding a `BuildRequires:  perl(ExtUtils::MakeMaker)` worked for me.
If you built using `mock`, from what i've noticed, anything that depends on `perl` does not give reliable results.
Can we please double check once.
PS: I could be wrong as well